x-pack/plugins/security_solution/server/assistant/tools/index.ts

```diff
@@ -22,4 +23,5 @@ export const getAssistantTools = (naturalLanguageESQLToolEnabled: boolean): Assi
   KNOWLEDGE_BASE_RETRIEVAL_TOOL,
   KNOWLEDGE_BASE_WRITE_TOOL,
   OPEN_AND_ACKNOWLEDGED_ALERTS_TOOL,
+  SECURITY_LABS_KNOWLEDGE_BASE_TOOL,
```

Comment:
I think we'll need to gate this behind the `assistantKnowledgeBaseByDefault` FF. Can either be done here, or in the tool itself via its `isSupported()` function.